CSE-Hub/practice questions/all testcases/README.md

```diff
@@ -0,0 +1 @@
+## reference link for .htaccess: [click here](https://www.thesitewizard.com/apache/password-protect-directory.shtml)
```

Comment:
It would be great if all these reference links are put in [resources](https://github.com/harshraj22/contest/blob/develop/resources/resources.md) as well. You might also want to add mysql injection there.